docs/samples/v1beta1/custom/simple.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: "serving.kubeflow.org/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "custom-simple"
+spec:
+  predictor:
+    containers:
+      - image: codait/max-object-detector
```

Comment:
Is this indented an extra layer too far?

Reply:
what do you mean by that?

Reply:
Maybe I'm crazy.
```    
containers:
- image: codait/max-object-detector
```
vs
```
containers:
  - image: codait/max-object-detector
```

Reply:
oh got it, good catch!